package/deb_postinst

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+# When installing the package, this script is called with the first argument
+# "configure". If the package is being upgraded from an earlier version, this version is
+# passed as the second argument.
+
+if [ "$1" = 'configure' ] && [ -n "$2" ]; then
+    # If upgrading from an earlier version than 0.13.1, the permissions on
+    # /etc/secrets.d and /usr/bin/vaultenv will be messed up.
+    # We fix this here:
+
+    if dpkg --compare-versions "$2" le 0.13.0; then
```

Comment:
Aren't the `chown` and `chmod` commands idempotent? Can we omit the version check (less code to verify the correctness of)?

Reply:
It should be, I've removed the `if` now.